src/compiler/reviews.md

```diff
@@ -291,7 +300,13 @@ refactorings, and no hard stability guarantees are promised.
 ### The PR is very large and complicated
 
 Reviewers are **not** expected to stomach PRs that are very large and
-complicated. Bring the PR to the attention of the team (through zulip threads
+complicated. It is expected from contributors to split their work as much as
+(reasonably) possible. Smaller, ideally self-contained bites make a review
+feasable. In general, before submitting large impact changes, it is expected the
```

Comment:
Suggestion: slightly adjust the wording here, maybe something like "a patch series of more digestible PRs which are individually more logically self-contained".

My thinking here is that micro PRs for the sake of small PRs isn't always good either. Could also recommend PR author discuss how to split the large change with the team as well.  Sometimes it might take different reviewers who are specialized/experts in different areas to review a subset of individual PRs in the patch series.